spec/factories/users.rb

```diff
@@ -7,6 +7,7 @@
     association :facility, strategy: :build
     device_updated_at { Time.now }
     device_created_at { Time.now }
+    sync_approval_status { User::SYNC_STATUSES.values.sample }
```

Comment:
Why not set this to a specific status? Looks like it would create some funky debugging situations if it's random.

Reply:
The factory is used to create random patients, so it made sense to also choose a random status. Agreed that it might make debugging a little difficult. We can be explicit when we need random statuses. Will change this in patient factory as well.